pycondor/utils.py

```diff
@@ -50,6 +49,9 @@ def _setup_logger(cls, verbose=0):
                        'Valid options are {}.'.format(
                            verbose, cls.name, logging_level_dict.keys()))
     logger = logging.getLogger(cls.name)
+    sh = logging.StreamHandler(sys.stdout)
```

Comment:
Nitpicky comment: can you give this a more descriptive name? Perhaps something like `stream_handler` instead of `sh`

Reply:
No problem.